File: .NET/potion-shop-k8s/Services/MessageService.cs
namespace potion_shop_k8s.Services
{
    /// <summary>
    /// Service for generating fun purchase messages
    /// </summary>
    public class MessageService
    {
        private readonly string[] _purchaseMessages = new[]
        {
            "You got a Red Potion! Your hearts are refilled!",
            "A fine purchase, brave hero! May it serve you well!",
            "One Red Potion coming right up! Stay safe out there!",
            "Excellent choice! This potion has saved many adventurers!",
            "A wise investment! You'll thank yourself later!",
            "Thank you for your patronage, noble warrior!",
            "It's dangerous to go alone! Take this potion!",
            "May this potion guide you to victory!"
        };

        /// <summary>
        /// Get a random purchase message
        /// </summary>
        public string GetRandomMessage()
        {
            var random = new Random();
            return _purchaseMessages[random.Next(_purchaseMessages.Length)];
        }
    }
}

File: Java/potion-shop-springboot/src/main/java/com/benson/potionshop/service/MessageService.java
package com.benson.potionshop.service;

import org.springframework.stereotype.Service;
import java.util.Random;

/**
 * Service for generating random purchase messages
 */
@Service
public class MessageService {

    private static final String[] PURCHASE_MESSAGES = {
        "You got a Red Potion! Your hearts are refilled!",
        "A fine purchase, brave hero! May it serve you well!",
        "One Red Potion coming right up! Stay safe out there!",
        "Excellent choice! This potion has saved many adventurers!",
        "A wise investment! You'll thank yourself later!",
        "Thank you for your patronage, noble warrior!",
        "It's dangerous to go alone! Take this potion!",
        "May this potion guide you to victory!"
    };

    private final Random random = new Random();

    public String getRandomMessage() {
        return PURCHASE_MESSAGES[random.nextInt(PURCHASE_MESSAGES.length)];
    }
}
